curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/63ff7976f760fe1d0b0fba4a.md

```diff
@@ -0,0 +1,146 @@
+---
+id: 63ff7976f760fe1d0b0fba4a
+title: Step 7
+challengeType: 0
+dashedName: step-7
+---
+
+# --description--
+
+Now, you will write a function to populate the authors on the UI. You will call this function inside the second `.then()` in the `fetch()` syntax.
+
+Write an arrow function `displayAuthors` and pass an `authors` parameter into it. This is where we'll do the DOM manipulation to display the authors. Use curly braces inside the arrow function because you'll need more than one expression.
```

Comment:
We will need to remove all references to first person narrative per the [docs](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=challenge-descriptionsinstructions) here